registry/directory/directory.go

```diff
@@ -42,12 +42,12 @@ import (
 	"github.com/apache/dubbo-go/remoting"
 )
 
-// Options ...
+// Options Option is configuration related  struct of serviceTTL
 type Options struct {
 	serviceTTL time.Duration
 }
 
-// Option ...
+// Option function of handling Options
 type Option func(*Options)
```

Comment:
what abount change to `OptionHandler` ?

Reply:
Option function of handling Options => Option will define a function of handling Options.